### DISCUSSION TITLE
Cannot figure out VOIP.MS

### DISCUSSION THREAD

**Discussion by forwardcomputers** [containrrr/shoutrrr#401] [Q&A]
I can send an SMS message via the VOIP.MS API using curl, eg:
`curl 'https://voip.ms/api/v1/rest.php?api_username=user@domain.com&api_password=$ecre!9&method=sendSMS&did=123456789&dst=987654321&message=Did+you+get+it'`

But I have not been able to figure it out using the generic service, any help is appreciated

**Comment by piksel** [comment]
There doesn't seem to be any public documentation for their API, but if it requires the message to be a query parameter I don't think it's possible to get it to work with the generic service.

**Reply by forwardcomputers** [reply]
Thank you for the response, it looks like it's a limitation on their API.

**Comment by forwardcomputers** [comment]
I was able to sent an SMS message, unfortunately VOIP.MS API needs the message as part of the URI.

Below worked
`generic://voip.ms/api/v1/rest.php?method=GET&api_username=user@domain.com&api_password=$ecre!9&__method=sendSMS&did=123456789&dst=987654321&message=Did+you+get+it`
